src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -501,7 +501,7 @@ protected override void OnNodePropertyChanged(object sender, PropertyChangedEven
 
             switch (e.PropertyName)
             {
-                case "IsUpdated":
+                case "CachedValue":
```

Comment:
This component is erroneously using `IsUpdated`, which could potentially cause it to initiate multiple rapid and potentially unnecessary updates.